docs/zh/latest/plugins/datadog.md

```diff
@@ -0,0 +1,145 @@
+---
+title: datadog
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## 简介
+
+`datadog` 是 Apache APISIX 内置的监控插件，可与 [Datadog](https://www.datadoghq.com/)（云应用最常用的监控和可观测性平台之一）无缝集成。`datadog` 插件支持对每个请求和响应周期进行多种指标参数的获取，这些指标参数基本反映了系统的行为和健康状况。
```

Comment:
```suggestion
`Datadog` 是 Apache APISIX 内置的监控插件，可与 [Datadog](https://www.datadoghq.com/)（云应用最常用的监控和可观测性平台之一）无缝集成。`Datadog` 插件支持对每个请求和响应周期进行多种指标参数的获取，这些指标参数基本反映了系统的行为和健康状况。
```
Maybe `Datadog` is better?  Is it better to keep a unified name？

Reply:
Agree, Thanks!

Reply:
> Agree, Thanks!

Because "datadog" is lowercase in config_default.yaml file, So I think it should be `datadog` instead of `Datadog`.

Reply:
But it's essentially a built-in plugin, and I'm confused about how to use the ' symbol.

Reply:
Thanks @hf400159  for the problem pointed out in time, I find it is a general style of APISIX’s documents, after taking look at other documents. 
So I will continue to use `datadog`. If you refer to other documents, you will find that every plugin’s name is surrounded with “`”.
@zaunist

Reply:
> Thanks @hf400159 for the problem pointed out in time, I find it is a general style of APISIX’s documents, after taking look at other documents. So I will continue to use `datadog`. If you refer to other documents, you will find that every plugin’s name is surrounded with “`”. @zaunist

Got it.